Makefile

```diff
@@ -284,33 +256,11 @@ test-cmd-debug: ## Run odo debug command tests
 	$(RUN_GINKGO) $(GINKGO_FLAGS) -focus="odo debug command tests" tests/integration/
 	$(RUN_GINKGO) $(GINKGO_FLAGS_SERIAL) -focus="odo debug command serial tests" tests/integration/debug/
 
-# Service, link and login/logout command tests are not the part of this test run
-.PHONY: test-integration
```

Comment:
I would say only remove `$(RUN_GINKGO) $(GINKGO_FLAGS_SERIAL) tests/integration/debug/` target because I can see some files are still targeting to `tests/integration/` test suite. For ex - `generic_test.go` , `cmd_perf_config_test.go` ...

Reply:
Fixed